packages/shared/lib/core/profile-manager/utils/buildProfileManagerOptionsFromProfileData.ts

```diff
@@ -1,20 +1,18 @@
 import { getStorageDirectoryOfProfile, IPersistedProfile } from '@core/profile'
 import { getSecretManagerFromProfileType, ProfileManagerOptions } from '@core/profile-manager'
-import { COIN_TYPE, getDefaultClientOptions } from '@core/network'
+import { COIN_TYPE, getDefaultClientOptions, NetworkProtocol } from '@core/network'
 
 export async function buildProfileManagerOptionsFromProfileData(
     profileData: Partial<IPersistedProfile>
 ): Promise<ProfileManagerOptions> {
-    const { id, networkProtocol, type } = profileData
+    const { id, type, network } = profileData
     const storagePath = await getStorageDirectoryOfProfile(id)
-    const coinType = COIN_TYPE[networkProtocol]
+    const coinType = COIN_TYPE[NetworkProtocol.Shimmer]
```

Comment:
This isn't `NetworkId`?

Reply:
Coin type still working of network protocol. This is to be refactored in another PR